extensions/kafka-client/runtime/src/main/java/io/quarkus/kafka/client/runtime/KafkaAdminClient.java

```diff
@@ -61,23 +61,23 @@ public Collection<ConsumerGroupDescription> getConsumerGroups() throws Interrupt
                 .values();
     }
 
-    public boolean deleteTopic(String name) {
+    public boolean deleteTopic(final String name) {
```

Comment:
I will let @phillip-kruger review this but could you get rid of all the added `final` modifiers?

Also make sure you squash all your commits when done.

Thanks!

Reply:
done!